### DISCUSSION TITLE
Where are the logs ?

### DISCUSSION THREAD

**Discussion by dorian33160** [pglombardo/PasswordPusher#2509] [Q&A] [answered]
Hi !

I am currently testing the app in a docker container. It is very good ! I saw on the documentation that there are logs for the interactions with the shared URL created by the app (maybe IP, etc.). If it exists, i am looking for the location.

Thank you for this amazing work.

**Comment by pglombardo** [comment, marked as answer]
Hi @dorian33160 - thanks for the nice words!

I believe you are referring to the Audit Logs.  To see those in the open source version, you have to [enable logins](https://docs.pwpush.com/docs/enabling-logins/) if you haven't already.

If you'd like a quicker way to see the audit logs, you can log into the opensource version of the application here:
https://oss.pwpush.com

And a few screenshots:
![pwpush 2021-09-12 23 41 40 127 0 0 1 7dd3815045bc](https://github.com/user-attachments/assets/56b8c534-3f87-4b4c-82d2-c4c73c0b0268)
 
![pwpush audit log 2021-09-28 20 00 39 pwpush com 6ecd3ef35656](https://github.com/user-attachments/assets/3025f5f4-a564-435e-8ec0-383fdb7e838c)

Hope this helps!

**Reply by dorian33160** [reply]
Thank you for the answer ! I didn't enable logins because I need an LDAP connection and as I saw in the requests that it is in your planing.
I'll wait for it cause I have too many users...
